SwanProjects/bin/swan_bash

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -i
```

Comment:
Why interactive (-i)?

Reply:
this is required because the bash session is interactive inside the SwanTerminal,
without this the prompt doesn't work properly.

Reply:
Ok!